Sources/pixelpass/pixelpass.swift

```diff
@@ -99,27 +107,39 @@ public class PixelPass {
         return nil
     }
     
-    public func getMappedCborData(jsonData: [String:String], mapper: [String:String]) -> [UInt8] {
-        var payload = [String: String]()
-        for (param, value) in jsonData {
-            let key = mapper[param] ?? param
-            payload[key] = value
-        }
-        let cborEncodableData = convertToCBOREncodableFormat(input: payload)
-        return cborEncodableData.encode()
+    public func getMappedData(stringData: String, mapper: [String:String], cborEnable : Bool = false) -> String {
```

Comment:
Reason to change return type from UInt8 to String is because it can be cborEncoded or normal data?

Reply:
Yes, the [UInt8]. was also eventually used after converting to String even in tests. And now the CBOR is optional it make sense to send Just the String back from this API.